addons/sale/report/sale_report_views.xml

```diff
@@ -35,10 +35,11 @@
                 <field name="user_id" optional="show" widget="many2one_avatar_user"/>
                 <field name="team_id" optional="show"/>
                 <field name="company_id" optional="show" groups="base.group_multi_company"/>
-                <field name="price_subtotal" optional="hide" sum="Sum of Untaxed Total"/>
-                <field name="price_total" optional="show" sum="Sum of Total"/>
+                <field name="price_subtotal" optional="hide" sum="Sum of Untaxed Total" widget="monetary"/>
+                <field name="price_total" optional="show" sum="Sum of Total" widget="monetary"/>
```

Comment:
Why is the widget spec necessary? If the fields is now a Monetary field, the widget used by framework should be monetary automatically already no?